packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -121,12 +133,14 @@ describe( 'Post Editor Performance', () => {
 				firstBlock,
 			} = await getLoadingDurations();
```

Comment:
Should this call be moved inside the "if" that follows?

Reply:
I was originally planning to keep things as close as possible between throwaway and real runs, but it doesn't really matter if the throwaway runs don't read the metrics. Moved.